frontend/packages/console-plugin-sdk/src/store.ts

```diff
@@ -62,24 +62,12 @@ export class PluginStore {
       ),
     );
     this.registry = new ExtensionRegistry(plugins);
-    this.updateDynamicExtensions = _.debounce(this.updateDynamicExtensions, 1000);
```

Comment:
FYI @spadgett 

The motivation was to debounce re-computations of Console extensions & subsequent invocation of all listeners, e.g. when a user fast-toggles a dynamic plugin's enabled status via UI.

However, reading up on debounce & throttle methods, I think it's best to debounce the actual UI event (e.g. click) handler, instead of doing it in plugin infra code.